app/src/org/commcare/android/tasks/ResourceEngineTask.java

```diff
@@ -149,16 +61,9 @@ public ResourceEngineTask(boolean upgradeMode, CommCareApp app,
     }
 
     @Override
-    protected ResourceEngineOutcomes doTaskBackground(String... profileRefs) {
+    protected AppInstallStatus doTaskBackground(String... profileRefs) {
         String profileRef = profileRefs[0];
-        AndroidCommCarePlatform platform = app.getCommCarePlatform();
-        SharedPreferences prefs = app.getAppPreferences();
-
-
-        // Make sure we record that an attempt was started.
-        Editor editor = prefs.edit();
-        editor.putLong(CommCarePreferences.LAST_UPDATE_ATTEMPT, new Date().getTime());
-        editor.commit();
+        ResourceInstallUtils.recordUpdateAttemptTime(app);

```

Comment:
Why does this count as an update attempt? Isn't ResourceEngineTask only for normal installations?